src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -1,62 +1,30 @@
 import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
 import { Entity } from "model/entity.model";
 
-// export const mapCompanyProfileToOverseasEntityToDTOUpdate = (data: CompanyProfile): Entity => {
-//   if (!data){
-//     return {} as Entity;
-//   }
-//   return {
-//     name: data?.companyName,
-//     dateOfCreation: data?.dateOfCreation,
-//     companyType: data?.type,
-//     companyNumber: data?.companyNumber,
-//     registeredOfficeAddress: mapCompanyAddressToOverseasEntityAddressDTO(data?.registeredOfficeAddress), // principal address
-//     // incorporationCountry: data?.foreign_company_details?.originating_registry?.country,
-//     // isOnRegisterInCountryFormedIn: mapYesOrNo(data),
-//     // serviceAddress: mapCompanyAddressToOverseasEntityAddressDTO(data?.registeredOfficeAddress),
-//     // legalForm: data?.legal_form,
-//     // lawGoverned: data?.foreign_company_details?.governed_by,
-//     // publicRegisterJurisdiction: data?.jurisdiction,
-//     // registrationNumber: data?.registration_number,
-//     // isServiceAddressSameAsPrincipalAddress: mapYesOrNo(data)
-//   };
-// };
-
 export const mapCompanyProfileToOverseasEntityToDTOUpdate = (cp: CompanyProfile): Entity => {
```

Comment:
rename  mapCompanyProfileToOverseasEntity() it's not a DTO. That part is done in the api SDK.